releasenotes/sf/README.md

```diff
@@ -21,9 +21,87 @@ Additional documentation:
+    * `sf package install report` : Retrieve the status of a package installation request.
+    * `sf package installed list` : List the org’s installed packages.
+    * `sf package list` : List all packages in the Dev Hub org.
+    * `sf package uninstall` : Uninstall a second-generation package from the target org.
+    * `sf package uninstall report` : Retrieve the status of a package uninstall request.
+    * `sf package update` : Update package details.
+    * `sf package version create` : Create a package version in the Dev Hub org.
+    * `sf package version create list` : List package version creation requests.
+    * `sf package version create report` : Retrieve details about a package version creation request.
+    * `sf package version delete` : Delete a package version.
+    * `sf package version displayancestry` : Display the ancestry tree for a 2GP managed package version.
+    * `sf package version list` : List all package versions in the Dev Hub org.
+    * `sf package version promote` : Promote a package version to released.
+    * `sf package version report` : Retrieve details about a package version in the Dev Hub org.
+    * `sf package version update` : Update a package version.
+
+    As always, run the commands with `--help` to see the list of flags, examples, and usage information. We'll be releasing other updated plugins over the next weeks. Enjoy!
+
+* NEW: The `sf org display` output now includes the API version of the org at the time you authorized it with the `sf login org` command. We cache the value locally, so if Salesforce updates your org to a new release, the API version will be incorrect. Re-login to your org to refresh the API version information in the `sf org display` output. (GitHub issue [#314](https://github.com/forcedotcom/cli/issues/314), plugin-org PR [#580](https://github.com/salesforcecli/plugin-org/pull/580))
+
+* NEW: (Bash and Zsh only) Use autocomplete on commands that use spaces by installing the [latest `plugin-autocomplete`](https://github.com/oclif/plugin-autocomplete/releases/tag/2.1.0):
+
+    ```bash
+    sf autocomplete
+    ```
+    Follow the displayed instructions to set it up in your environment. Then use the tab key to autocomplete commands as usual.  But wait, you get these new features too:
+    
+    * Type `--` to get suggestions for flags that define a [known set of options](https://oclif.io/docs/flags).
+    * Flags that can be specified multiple times are still suggested, even if you've already used it. 
+    * Type `-` to get suggestions of short flag names.
+    
+    If you currently use autocomplete for colon-separated commands, nothing changes in your environment until you explicitly update to the latest `plugin-autocomplete` version. If you updated, but want to go back to autocompleting commands that use `:` as a separator, set this environment variable:
+    
+    ```bash
+    OCLIF_AUTOCOMPLETE_TOPIC_SEPARATOR=colon
+    ```
```

Comment:
after setting this env var the autocomplete cache needs to be re-generated by running `sf autocomplete --refresh-cache`